### PR TITLE
add wikidata/wikipedia for 8 banks

### PR DESCRIPTION
**Pull request by Sukkoria**
- Московский индустриальный банк #333
- Мособлбанк #334
- ПУМБ #337
- Совкомбанк #339
- Укргазбанк #340
- Уралсиб #341
- Хаан банк #342
- بانک شهر 
 #346

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -4268,6 +4276,7 @@
   },
   "amenity/bank|Ощадбанк": {
     "count": 958,
+    "countryCodes": ["ua"],
     "tags": {
       "amenity": "bank",
       "brand": "Ощадбанк",
```
Comment (eigenbrot) [inline config/canonical.json]: Are the following wiki entries not correct?
https://www.wikidata.org/wiki/Q4340839
https://uk.wikipedia.org/wiki/%D0%9E%D1%89%D0%B0%D0%B4%D0%B1%D0%B0%D0%BD%D0%BA

File: config/canonical.json [hunk 2]
```diff
@@ -4464,14 +4486,21 @@
...
+      "name:en": "Uralsib bank"
     }
   },
   "amenity/bank|Хаан банк": {
     "count": 52,
+    "countryCodes": ["mn"],
     "tags": {
       "amenity": "bank",
       "brand": "Хаан банк",
+      "brand:wikidata": "Q25632240",
```
Comment (eigenbrot) [inline config/canonical.json]: This wikidata entry is really bad, but I suppose there's nothing we can do about that

File: config/canonical.json (hunk 1 above)
Comment (Sukkoria) [reply config/canonical.json]: I had not found this one, but they seem correct, indeed.

File: config/canonical.json (hunk 2 above)
Comment (Sukkoria) [reply config/canonical.json]: I just added "instance of : bank" and "country : Mongolia", but I couldn’t infer much more from the wikipedia article :/

Comment (eigenbrot) [reply config/canonical.json]: Nice!

File: config/canonical.json (hunk 1 above)
Comment (eigenbrot) [reply config/canonical.json]: OK, just push a commit with the changes and I'll approve the PR.

Comment (Sukkoria) [reply config/canonical.json]: That should be done in commit 8316ffb ? Thanks for the help !

**Comment by eigenbrot** [comment]
Looks great. Thanks @Sukkoria!

**Pull request merged**